installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.adoc

```diff
@@ -27,9 +27,11 @@ include::modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc[leve
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../../post_installation_configuration/bare-metal-configuration.adoc[Bare metal configuration]
+* xref:../../backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.adoc#replacing-the-unhealthy-etcd-member[Replacing an unhealthy etcd member]
+
+* xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backing-up-etcd-data_backup-etcd[Backing up etcd]
 
-* xref:../../backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.adoc#restore-replace-stopped-etcd-member_replacing-unhealthy-etcd-member[Replacing an unhealthy etcd member]
+* xref:../../post_installation_configuration/bare-metal-configuration.adoc[Bare metal configuration]
```

Comment:
Sorry, I missed this earlier, but this xref needs an #anchor specified after the .adoc